scripts/update-status-go.sh

```diff
@@ -74,6 +75,7 @@ fi
 
 # ls-remote finds only tags, branches, and pull requests, but can't find commits
 STATUS_GO_MATCHING_REFS=$(git ls-remote ${REPO_URL} ${STATUS_GO_VERSION})
+STATUS_GO_COMMIT_MESSAGE=$(curl $(curl ${REPO_API_URL}/git/ref/tags/${STATUS_GO_VERSION} | jq -r .object.url) | jq -r .message)
```

Comment:
Why would you assume the ref provided as argument is always a tag?

Reply:
Now I figured out what is really to be done and will fix soon...

Reply:
Calling GitHub API is a bit ugly, but unfortunately we cannot get commit messages from `git ls-remote`.

Reply:
Since we have to hanndle multiple types of objects - commits, branches, tags -  this code will have to be more complex than it is right now, which is a bit annoying.

Reply:
Also, small note, if you just call `curl` without any flags it will fail silently, without an error return code.
You can provide `-f`/`--fail` to make it return an error, if that's what we actually want.
The other thing to possibly use is `-sS`, to make it silent, but show errors. And possibly `-L` for redirects.

Reply:
One possible way to do this without calling the API is to fetch the object into the local repo(`status-react`) one:
```sh
~/work/status-react develop
 > g fetch --depth=1 status-go develop
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
From https://github.com/status-im/status-go
 * branch                develop    -> FETCH_HEAD

~/work/status-react develop
 > g log --format=format:'%s' status-go/develop
load audio from http server
```
Which is also a bit ugly, but we could clean up after that. Also, need to check how that works with tags.